R/miic.R

```diff
@@ -444,6 +449,7 @@ miic <- function(input_data,
 
   is_contextual <- rep(0, ncol(input_data))
   names(is_contextual) <- colnames(input_data)
+  is_consequence <- is_contextual
```

Comment:
If I want to be really nit-picking, I would prefer to have a "blanck_vector <- rep(0, ncol(input_data)) etc ... and then is_contextual <- blanck_vector; is_consequence <- blanck_vector" for clarity of variable names (as "is_consequence <- is_contextual" is misleading if not careful) 
But not a necessary change

Reply:
I agree with you, the two lines init would be more self explicative. 
If this is the only change, I would suggest that we go over. 
We can do some cosmetic changes later.